<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>runLiveScecne</class>
 <widget class="QWidget" name="runLiveScecne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="processNameTextBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="burstTimeSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Process Details</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Burst Time</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Priority</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="addLiveProcessButton">
        <property name="text">
         <string>Add Process Live</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="prioritySpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="2">
       <widget class="QTableWidget" name="processStatsTable">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <column>
         <property name="text">
          <string>PID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Remaining Burst Time</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Waiting Time</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Turnaround Time</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Response Time</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="runAllButton">
        <property name="text">
         <string>Run All</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pauseButton">
        <property name="text">
         <string>Pause / Resume</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Process Stats</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Gantt Chart</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="runLiveButton">
        <property name="text">
         <string>Run Live</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QFrame" name="ganttPlaceHolder">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="returnToInputSceneButton">
        <property name="text">
         <string>Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
